mercuryseries/inertia-bundle/0.1/config/routes/inertia.yaml

```diff
@@ -0,0 +1,19 @@
+# If you use PHP attributes
+# controllers:
+#     resource:
+#         path: ../../src/Controller/
+#         namespace: App\Controller
+#     type: attribute
+#     defaults:
+#         csrf:
+#             create: true
+#             require: ['POST', 'PUT', 'PATCH', 'DELETE']
+
+# If you use annotations
+# controllers:
+#     resource: ../../src/Controller/
+#     type: annotation
+#     defaults:
+#         csrf:
+#             create: true
+#             require: ['POST', 'PUT', 'PATCH', 'DELETE']
```

Comment:
You only need one of these - iirc, `type: annotations` will work for both.

However, if I'm understanding this correctly, then what you are REALLY doing is just adding the `defaults: csrf:` part to the user's existing route import, right? In that case, my previous comment was wrong... and you may indeed need to use `add-lines` to add this to the existing `config/routes.yaml`.

Reply:
Thanks @weaverryan for the feedback. Really appreciated!
I finally decided to handle this on the bundle side. So I think it can be merged now.